### PR TITLE
Fix tests and rename the key for project init

### LINKED ISSUE #1258: Fix broken e2e test on Windows
Was introduced by #1254
@anayden, please fix ASAP

### PR DESCRIPTION
**Pull request by anayden**
Closes #1258

### CODE REVIEW

File: tests/e2e/test_project.py [hunk 1]
```diff
@@ -10,21 +10,25 @@
...
+    assert path.exists()
+    assert path.is_dir()
     try:
-        helper.run_cli(["project", "init", "-q"])
-        assert path.exists()
-        assert path.is_dir()
-    finally:
         shutil.rmtree(path)
+    except Exception:
+        # ignore errors, the directory may not exist legitimately
```
Comment (serhiy-storchaka) [inline tests/e2e/test_project.py]: If the directory may not exist then `assert path.is_dir()` can fail?

File: tests/e2e/test_project.py [hunk 2]
```diff
@@ -11,20 +11,22 @@ def test_project_init(helper: Helper) -> None:
     path = pathlib.Path(f"./name-of-the-project")
     assert not path.exists()
     try:
-        helper.run_cli(["project", "init", "-q"])
+        helper.run_cli(["project", "init", "--no-input"])
         assert path.exists()
         assert path.is_dir()
     finally:
-        shutil.rmtree(path)
+        if path.exists():
```
Comment (serhiy-storchaka) [inline tests/e2e/test_project.py]: It still does not look right. If `path.exists()` can be false then `assert path.exists()` will fail. If remove asserts, we will loss the way to check that `project init` works.

Comment (anayden) [reply tests/e2e/test_project.py]: What I observe now is that `project init` fails with exception on Windows. I'm investigating that first.

**Comment by ayushkovskiy** [comment]
Let's close this PR in favour of https://github.com/neuromation/platform-client-python/pull/1260.

**Comment by anayden** [comment]
Done

**Pull request closed without merging**